upstream-example.Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN /bin/initializer -o ./bundles.db
 
 FROM scratch
 COPY ["nsswitch.conf", "/etc/nsswitch.conf"]
-COPY --from=builder /build/bundles.db /bundles.db
```

Comment:
Removing `/build` can break this for people using an older version of podman. 
A better fix will be adding `WORKDIR /build` on Line 2